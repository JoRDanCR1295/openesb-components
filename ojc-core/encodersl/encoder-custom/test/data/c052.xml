<?xml version="1.0" encoding="UTF-8"?><root xmlns="http://xml.netbeans.org/schema/test" xsi:schemaLocation="${SCHEMA_LOCATION}" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><HEADElement><HEADId>HEAD</HEADId><Ctypalgo>DP</Ctypalgo><Ccosys>CG</Ccosys><Cdealn>20</Cdealn><Rkontr>A771923</Rkontr><Rvers>AA</Rvers><Cprodu>D </Cprodu><Source>IBS_FX/MM</Source><Rundate>20080204</Rundate><Runtime>080548</Runtime></HEADElement><DEALElement><DEALId>DEAL</DEALId><Ctypalgo>DP</Ctypalgo><Ccosys>CG</Ccosys><Cdealn>20</Cdealn><Rkontr>A771923</Rkontr><Rvers>AA</Rvers><Cbeen>N</Cbeen><Targetsystem></Targetsystem><Rcifd>672073</Rcifd><Rprof>2</Rprof><Cocomp>CGBO</Cocomp><Cprodu>D </Cprodu><Ccorr>N</Ccorr><Copera>L</Copera><Dkontr>20080204</Dkontr><Dregi>20080204</Dregi><Dureg>80548</Dureg><Dvaldl>20080204</Dvaldl><Dmatdl>20080221</Dmatdl><Cdealr>XX</Cdealr><Cusrid>OLK     </Cusrid><Rtegp>801633</Rtegp><Rbrokr>0</Rbrokr><Cwijze>I</Cwijze><Ccashfl>Y</Ccashfl><Cinfo>N</Cinfo><Cerror>000000</Cerror><Cmuhm>USD</Cmuhm><Bbedhm>-80230.07</Bbedhm><Cmutm></Cmutm><Bbedtn>+0.00</Bbedtn><Cmualf>USD</Cmualf><Cmt_intr></Cmt_intr><Bnot>-80230.07</Bnot><Cbasis>B</Cbasis><Cbuss>SM</Cbuss><Cintc>1Y</Cintc><Cintb>1Y</Cintb><Cintk>N</Cintk><Cintt>F</Cintt><Ckintr>N</Ckintr><Bbedtf>+0.00</Bbedtf><Bratsp>+0.0000000000</Bratsp><Bratfk>+0.0000000000</Bratfk><Bratnd>+0.0000000000</Bratnd><Bratfd>+0.0000000000</Bratfd><Brati>+3.0340000000</Brati><Bfldev>+0.0000000000</Bfldev><Dfcoup>0</Dfcoup><Camor>N</Camor><Cntwkr>N </Cntwkr><Cffsrt>N   </Cffsrt><Reffid></Reffid><Nwde></Nwde><Cmuwde></Cmuwde><Bnomst>+0.00</Bnomst><Duitg>0</Duitg><Dterug>0</Dterug><Reffas></Reffas><Aeenhd>0</Aeenhd><Bfixr>+3.0340000000</Bfixr><Cswfc>C</Cswfc><Cswter>N</Cswter><Cfkrs_ref3></Cfkrs_ref3><Cfkrs_ref_det3></Cfkrs_ref_det3><Cloopt_fkrs></Cloopt_fkrs><Aopzeg>0</Aopzeg><Ciber>A</Ciber><Cswct>D</Cswct><Cswie>D</Cswie><Cswre>D</Cswre><Ddatfix>0</Ddatfix><Ddatspc>0</Ddatspc><Cmubegfee></Cmubegfee><Bbegfee>+0.00</Bbegfee><Cmuendfee></Cmuendfee><Bendfee>+0.00</Bendfee><Coddfstcoup>N</Coddfstcoup><Lstregcoup>0</Lstregcoup><Assetstartdate>0</Assetstartdate><Bcurrate>+0.0000000000</Bcurrate><Bnxtrate>+0.0000000000</Bnxtrate><Bdragl>+0.00</Bdragl><Dbcurrateb>0</Dbcurrateb><Dbcurratee>0</Dbcurratee><Dbnxtrateb>0</Dbnxtrateb><Dbnxtratee>0</Dbnxtratee><Spotdate>0</Spotdate><Spotrate>+0.0000000000</Spotrate><PC_Cmualf_1></PC_Cmualf_1><PC_Cmualf_2></PC_Cmualf_2><PC_Cmualf_3></PC_Cmualf_3><PC_Cmualf_4></PC_Cmualf_4><Csirw>N</Csirw><Ext_System>MX</Ext_System><Ext_Reference>1813783                            </Ext_Reference></DEALElement><CASHElement><CASHId>CASH</CASHId><Ccosys>CG</Ccosys><Cdealn>20</Cdealn><Rkontr>A771923</Rkontr><Rvers>AA</Rvers><Cbeen>N</Cbeen><Csrtbe>K</Csrtbe><Datbet>20080204</Datbet><Ddateb>20080204</Ddateb><Ddatee>20080221</Ddatee><Cprodu>D </Cprodu><Cmualf>USD</Cmualf><Bbetal>-80230.07</Bbetal><Breken>+80230.07</Breken><Brent>+3.0340000000</Brent><Bfix>+0.0000000000</Bfix><Cficre>R</Cficre></CASHElement><CASHElement><CASHId>CASH</CASHId><Ccosys>CG</Ccosys><Cdealn>20</Cdealn><Rkontr>A771923</Rkontr><Rvers>AA</Rvers><Cbeen>N</Cbeen><Csrtbe>K</Csrtbe><Datbet>20080221</Datbet><Ddateb>20080204</Ddateb><Ddatee>20080221</Ddatee><Cprodu>D </Cprodu><Cmualf>USD</Cmualf><Bbetal>+80230.07</Bbetal><Breken>+80230.07</Breken><Brent>+3.0340000000</Brent><Bfix>+0.0000000000</Bfix><Cficre>R</Cficre></CASHElement><MEMOElement><MEMOId>MEMO</MEMOId><Rseq></Rseq><Cmed>I</Cmed><Tmed>TMB_005326752_1_1                  </Tmed></MEMOElement><MEMOElement><MEMOId>MEMO</MEMOId><Rseq></Rseq><Cmed>I</Cmed><Tmed>2008-02-02;+;3.03201;010000        </Tmed></MEMOElement><MEMOElement><MEMOId>MEMO</MEMOId><Rseq></Rseq><Cmed>I</Cmed><Tmed>000004901933;101                   </Tmed></MEMOElement><TRLRElement><TRLRId>TRLR</TRLRId><Ccosys>CG</Ccosys><Cdealn>20</Cdealn><Rkontr>A771923</Rkontr><Rvers>AA</Rvers><Nrec>008</Nrec></TRLRElement></root>