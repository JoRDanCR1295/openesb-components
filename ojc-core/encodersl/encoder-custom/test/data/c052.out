HEAD,DP,CG,20,A771923,AA,D ,IBS_FX/MM,20080204,080548;DEAL,DP,CG,20,A771923,AA,N,672073,2,CGBO,D ,N,L,20080204,20080204,80548,20080204,20080221,XX,OLK     ,801633,0,I,Y,N,000000,USD,-80230.07,+0.00,USD,-80230.07,B,SM,1Y,1Y,N,F,N,+0.00,+0.0000000000,+0.0000000000,+0.0000000000,+0.0000000000,+3.0340000000,+0.0000000000,0,N,N ,N   ,,,+0.00,0,0,0,+3.0340000000,C,N,,,0,A,D,D,D,0,0,+0.00,+0.00,N,0,0,+0.0000000000,+0.0000000000,+0.00,0,0,0,0,0,+0.0000000000,,,,N,MX,1813783                            ;CASH,CG,20,A771923,AA,N,K,20080204,20080204,20080221,D ,USD,-80230.07,+80230.07,+3.0340000000,+0.0000000000,R;CASH,CG,20,A771923,AA,N,K,20080221,20080204,20080221,D ,USD,+80230.07,+80230.07,+3.0340000000,+0.0000000000,R;MEMO,I,TMB_005326752_1_1                  ;MEMO,I,2008-02-02;+;3.03201;010000        ;MEMO,I,000004901933;101                   ;TRLR,CG,20,A771923,AA,008